### PR TITLE
Fix podspec

### PR DESCRIPTION
**Pull request by tattn**
I fixed deployment target and resources.

### CODE REVIEW

**Comment by taisukeh** [comment]
Thanks for the PR @tattn .

When I tried `pod install` with a sample app, i noticed there are following problems.

### All header files are copied as public 

I think you need to use `public_header_files`.
https://guides.cocoapods.org/syntax/podspec.html#public_header_files
> If no public headers are specified then all the headers in source_files are considered public.

### `Main.storyboard` overwrite client project's storybard

https://guides.cocoapods.org/syntax/podspec.html#resources
> Moreover, resources specified with this attribute are copied directly to the client target

So this SDK will break client app.

### Others

This is not a problem, but I think that it is better to specify `platform`. Leaving `platform` blank means the Pod is supported on all platforms.

https://guides.cocoapods.org/syntax/podspec.html#platform

**Comment by taisukeh** [comment]
I would appreciate if you try use the podspec to resove problems :bow:

**Comment by tattn** [comment]
Oh, sorry. You have a point!

I changed `resources` to `resource_bundles`.

Installing by CocoaPods, it uses `AppFeedbackResource.bundle`.
Installing by Carthage, it uses resources in the generated framework.

**Comment by taisukeh** [comment]
Thanks @tattn , looks great! Could you add `Localizable.strings` files to the `resource_bundles` ?

**Comment by tattn** [comment]
Thanks for your review :-)
I added it!

File: AppFeedback.podspec [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
-  s.homepage     = "https://github.com/yahoojapan/AppFeedback.git"
-  s.license      = "MIT"
-  s.author       = "Yahoo Japan Corporation"
+  s.homepage              = "https://github.com/yahoojapan/AppFeedback.git"
+  s.license               = "MIT"
+  s.author                = "Yahoo Japan Corporation"
 
-  s.ios.deployment_target = "8.0"
+  s.ios.deployment_target = "9.0"
+  s.platform = :ios, '9.0'
```
Comment (taisukeh) [inline AppFeedback.podspec]:
```suggestion
```

Sorry it was my misunderstanding. It was no problem if `ios.deployment_target` was specified.

Comment (taisukeh) [reply AppFeedback.podspec]: ( Please correct it again if there is a problem. )

**Review by taisukeh — APPROVED** [review]
Thanks! 👍

File: AppFeedback.podspec (hunk 1 above)
Comment (tattn) [reply AppFeedback.podspec]: Thanks

**Pull request merged**